joplin/pages/base_page/models.py

```diff
@@ -53,20 +53,21 @@ def janis_urls(self):
         """
         This should handle coa_global and department stuff
         """
+        branch_settings = JanisBranchSettings.objects.first()
 
         # If we're global, even if we have a department, we should only exist at
         # /page_slug
         # and not at
         # /department_slug/page_slug
         if self.coa_global:
-            return ['{base_url}{page_slug}/'.format(base_url=self.janis_url_base('publish_janis_branch'),
+            return ['{base_url}{page_slug}/'.format(base_url=branch_settings.get_publish_url_base(),
```

Comment:
now I'm wondering what `janis_url_base('publish_janis_branch)` was doing before, this seems smarter